numpy/linalg/linalg.py

```diff
@@ -930,7 +930,11 @@ def eigvalsh(a, UPLO='L'):
         computed.
     UPLO : {'L', 'U'}, optional
         Same as `lower`, with 'L' for lower and 'U' for upper triangular.
-        Deprecated.
+        Deprecated. 
```

Comment:
I have no recollection of this, but 74e14775a144794 mentions `lower` so maybe the idea was to harmonize this with scipy's kwargs and replace the U/L with lower=True/False kwarg.